tools/ci_build/github/azure-pipelines/linux-gpu-cuda-10-pipeline.yml

```diff
@@ -0,0 +1,64 @@
+jobs:
```

Comment:
We don't need a new CUDA 10.2 pipeline anymore. We want it becoming the default.